migration/idempotent/011-maintenance.sql

```diff
@@ -648,6 +648,122 @@ BEGIN
+            FROM x
+            UNION ALL
+            SELECT format('GRANT ALL PRIVILEGES ON TABLE %I.%I TO %I', x.schema_name, x.table_name, quote_ident(session_user))
+            FROM x
+        )
+        LOOP
+            EXECUTE _sql;
+        END LOOP
+        ;
+
+        -- indexes on compressed tables
+        FOR _sql IN
+        (
+            SELECT format('ALTER TABLE %I.%I OWNER TO prom_admin', h2.schema_name, h2.table_name)
+            FROM _timescaledb_catalog.hypertable h1
+            INNER JOIN _timescaledb_catalog.hypertable h2
+            ON (h1.schema_name = ANY(ARRAY['prom_data', '_ps_trace'])
+            AND h1.compressed_hypertable_id = h2.id)
+        )
+        LOOP
+            EXECUTE _sql;
+        END LOOP
+        ;
+
+        -- compressed tables' chunks
+        FOR _sql IN
+        (
+            SELECT format('ALTER TABLE %I.%I OWNER TO prom_admin', k.schema_name, k.table_name)
+            FROM _timescaledb_catalog.hypertable h1
+            INNER JOIN _timescaledb_catalog.hypertable h2
+            ON (h1.schema_name = ANY(ARRAY['prom_data', '_ps_trace'])
+            AND h1.compressed_hypertable_id = h2.id)
+            INNER JOIN _timescaledb_catalog.chunk k
+            ON (h2.id = k.hypertable_id)
+        )
+        LOOP
+            EXECUTE _sql;
+        END LOOP
+        ;
+
+        -- hypertable chunks
+        FOR _sql IN
+        (
+            WITH x AS
+            (
+                SELECT
+                  k.schema_name
+                , k.table_name
+                FROM _timescaledb_catalog.hypertable h
+                INNER JOIN _timescaledb_catalog.chunk k
+                ON (h.id = k.hypertable_id)
+                WHERE h.schema_name = ANY(ARRAY['prom_data', '_ps_trace'])
+            )
+            SELECT format('ALTER TABLE %I.%I OWNER TO prom_admin', x.schema_name, x.table_name)
+            FROM x
+        )
+        LOOP
+            EXECUTE _sql;
+        END LOOP
+        ;
+/*
+        FOR _sql IN
+        (
+            WITH x AS
+            (
+                SELECT
+                  k.schema_name
+                , k.table_name
+                FROM _timescaledb_catalog.hypertable h1
+                INNER JOIN _timescaledb_catalog.hypertable h2
+                ON (h1.schema_name = ANY(ARRAY['prom_data', '_ps_trace'])
+                AND h1.compressed_hypertable_id = h2.id)
+                INNER JOIN _timescaledb_catalog.chunk k
+                ON (h.id = k.hypertable_id)
+            )
+            SELECT format('GRANT ALL PRIVILEGES ON TABLE %I.%I TO prom_admin', x.schema_name, x.table_name)
+            FROM x
+            UNION ALL
+            SELECT format('ALTER TABLE %I.%I OWNER TO prom_admin', x.schema_name, x.table_name)
+            FROM x
+            UNION ALL
+            SELECT format('GRANT ALL PRIVILEGES ON TABLE %I.%I TO %I', x.schema_name, x.table_name, quote_ident(session_user))
```

Comment:
This has the same issue as the previous one with `%I` and `quote_ident`. Perhaps worth fixing in case this is ever uncommented.